strongbox-commons/src/main/resources/application.yaml

```diff
@@ -24,21 +24,23 @@ strongbox:
       feed: false
   version: @{project.version}
   revision: @{strongbox.revision}
-  orientdb:
-    studio:
-      enabled: false
-      ip:
-        address: 127.0.0.1
-      port: 2480
-  server:
-    database:
-      path: ${strongbox.vault}/db
+  db:
+    janus-graph:
+      storage-username: cassandra
+      storage-password: cassandra
```

Comment:
These credentials should be the same as for Strongbox itself.

Reply:
```suggestion
      # TODO: [First start] randomize
      storage-username: admin
      storage-password: password
```

Reply:
this is not a trivial thing to change default credentials for embedded Cassandra. Will create a follow up task so someone can research how to do it properly.

Reply:
[issues/1843](https://github.com/strongbox/strongbox/issues/1843)